### PR TITLE
Docs: Clarify relative time range overrides

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
We recently received an issue related to the usage of relative time overrides in panels for a dashboard with a time range set to an absolute time range. This PR tries to clarify this a bit.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Friendly reminder to myself not to backport this before 8.3.2 is released 👍

### CODE REVIEW

File: docs/sources/dashboards/time-range-controls.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ For more advanced time settings, click the **Dashboard settings** (gear) icon at
 
 In [Query options]({{< relref "../panels/queries.md#query-options" >}}), you can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the upper right. This allows you to show metrics from different time periods or days at the same time.
 
+> **Note:** Panel time overrides have no effect when the time range for the dashboard is set to an absolute time range.
```
Comment (achatterjee-grafana) [inline docs/sources/dashboards/time-range-controls.md:115]:
```suggestion
> **Note:** Panel time overrides have no effect when the time range for the dashboard is absolute.
```

File: docs/sources/panels/queries.md [hunk 2]
```diff
@@ -95,6 +95,8 @@ Panel data source query options:
 
 - **Relative time -** You can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the top right corner of the dashboard. This allows you to show metrics from different time periods or days on the same dashboard.
 
+  > **Note:** Panel time overrides have no effect when the time range for the dashboard is set to an absolute time range.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/queries.md:98]:
```suggestion
  > **Note:** Panel time overrides have no effect when the time range for the dashboard is absolute.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

**Review by ashharrison90 — APPROVED** [review]
lgtm 🙌 
might be worth checking with simon as there's some overlap between this and https://github.com/grafana/grafana/pull/42884

**Comment by hugohaggmark** [comment]
Thanks @achatterjee-grafana!

**Pull request merged**